syntax = "proto3";

package videostreaming;

// 비디오 프레임 메시지 정의
message VideoFrame {
  bytes frame = 1;
}

// 비디오 요청 메시지 정의
message VideoRequest {
  string video_name = 1;
}

// 서비스 정의
service VideoStreamingService {
  // 서버에서 클라이언트로 스트리밍 RPC
  rpc StreamVideo(VideoRequest) returns (stream VideoFrame);
}
